todo.py

```diff
@@ -3,19 +3,41 @@ def print_todos(todos):
     if len(todos) == 0:
         print("You have nothing to do.")
     else:
-        i = 0
+        # i = 0
+        completed_list = []
```

Comment:
Nice! Clear approach to printing out the different categories of todos (complete vs incomplete). One question, since you reset the `count` to `0` before you print each category of todo, how does the user know the actual index of any single todo? (That is, if they want to set a todo as completed, how do they know what index to specify?)